### ISSUE TITLE
Frontend Filter: getDivision() must be of the type array, null returned 

### ISSUE THREAD

**Issue by peerlight** [itx-informationssysteme/jobapplications#44]
When filtering list view this error occurs

Return value of ITX\Jobapplications\Domain\Model\Constraint::getDivision() must be of the type array, null returned 

TYPO3 9.5.28
jobapplications 1.0.2

**Comment by peerlight** [comment]
This was due to a misunderstanding of how the filters can be tweaked.

It is not possible to just comment out filterOptions in FLUID. So I just hide not needed filterOptions ...

Can be closed or transfered to a feature request: Make filterOptions configure-/switchable

**Comment by itx-benjamin-jasper** [comment]
I agree, this behavior is not very intuitive. Basically you would have to empty the filter config array in Typoscript. A switch is one way, but I think a better way would be to treat a request without filters as empty filters and simply return the unfiltered result.
